azure-pipelines.yml

```diff
@@ -425,53 +87,14 @@ jobs:
           testResultsFormat: 'JUnit'
           testResultsFiles: '**/e2e-tests-report*.xml'
         condition: succeededOrFailed()
-  # - job: E2e_Test_TeamsV1
-  #   displayName: 'E2e test TeamsJS V1'
+  # - job: E2ETest4
```

Comment:
I think this is now enabled in 2.0-preview (a merge should bring that change in).

Reply:
Fixed